Libraries/LibWeb/CSS/MediaQuery.h

```diff
@@ -73,20 +77,20 @@ class MediaFeatureValue {
         return m_value.get<Ratio>();
     }
 
-    Resolution const& resolution() const
+    ResolutionOrCalculated const& resolution() const
     {
         VERIFY(is_resolution());
-        return m_value.get<Resolution>();
+        return m_value.get<ResolutionOrCalculated>();
     }
 
-    float number() const
+    IntegerOrCalculated integer() const
```

Comment:
What's the reasoning with changing number -> integer? The spec is a bit weird but does [say:](https://drafts.csswg.org/mediaqueries-5/#typedef-mf-value)
> `<mf-value> = <number> | <dimension> | <ident> | <ratio>`

Though, all the actual media features are defined as `<integer>` so yeah it's odd. Still, is this something you needed for your code to work?

Reply:
I don't know if it was absolutely necessary, but the reason I changed it is because the parser specifically parses only integers, and creates an `IntegerOrCalculated`. So this was the easiest way I could think of to pass that to the `MediaFeatureValue`, because it only ever uses integers anyway.